python-sdk/nuscenes/eval/predict/data_classes.py

```diff
@@ -0,0 +1,38 @@
+from typing import List, Callable, Dict, Any
+import dataclasses
+import numpy as np
+from nuscenes.eval.common.data_classes import MetricData
+
+@dataclasses.dataclass
+class Prediction(MetricData):
+    """Stores predictions of Models.
+    Metrics are calculated from Predictions.
+    :param instance: Instance token for prediction.
```

Comment:
I think :param: can only be used for method arguments, not class attributes.

Reply:
Removed